package.json

```diff
@@ -2,23 +2,55 @@
   "name": "node-minizip",
   "version": "0.1.0",
   "description": "A node.js addon binding Minizip for handling zip files.",
-  "keywords": ["zip", "minizip", "zlib", "unzip"],
+  "keywords": [
+    "zip",
+    "minizip",
+    "zlib",
+    "unzip"
+  ],
   "main": "main.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/hokein/node-minizip"
   },
-  "licenses": [{
-    "type": "MIT",
-    "url": "https://github.com/hokein/node-minizip/blob/master/LICENSE"
-  }],
+  "licenses": [
+    {
+      "type": "MIT",
+      "url": "https://github.com/hokein/node-minizip/blob/master/LICENSE"
+    }
+  ],
   "bugs": {
     "url": "https://github.com/hokein/node-minizip/issues"
   },
   "dependencies": {
-    "nan": "1.3.0"
+    "nan": "^2.0.5"
   },
   "devDependencies": {
     "mocha": "1.21.4"
-  }
+  },
+  "scripts": {
+    "install": "node-gyp rebuild"
+  },
+  "gypfile": true,
+  "homepage": "https://github.com/hokein/node-minizip",
+  "_id": "node-minizip@0.1.0",
+  "_shasum": "f4eccd07bd590fd5de061fc42886002b0decef05",
+  "_from": "node-minizip@*",
+  "_npmVersion": "1.4.9",
+  "_npmUser": {
```

Comment:
I'm not very familar with the fields in `npm` package.json. 
Are the fields with `_` prefix needed? From my experience, I didn't find these fields in other npm modules.